package.json

```
@@ -21,22 +21,22 @@
 		"url": "https://github.com/minio/kes-docs-theme/issues"
 	},
 	"devDependencies": {
-		"@tailwindcss/typography": "^0.5.10",
-		"autoprefixer": "^10.4.17",
-		"concurrently": "^8.2.2",
-		"prettier": "^3.2.5",
+		"@tailwindcss/typography": "^0.5.15",
+		"autoprefixer": "^10.4.20",
+		"concurrently": "^9.1.0",
+		"prettier": "^3.3.3",
 		"prettier-plugin-go-template": "^0.0.15",
-		"prettier-plugin-tailwindcss": "^0.5.11"
+		"prettier-plugin-tailwindcss": "^0.6.8"
 	},
 	"browserslist": [
 		"last 2 versions"
 	],
 	"dependencies": {
-		"@cfworker/json-schema": "^1.12.8",
+		"@cfworker/json-schema": "^2.0.1",
 		"clipboard": "^2.0.11",
-		"esbuild": "^0.20.1",
-		"flexsearch": "0.7.31",
+		"esbuild": "^0.24.0",
+		"flexsearch": "0.7.43",
```

```suggestion
		"flexsearch": "0.7.31",
```

Without this we get:

```
> esbuild --bundle --sourcemap --target=es2018 --outfile=./assets/dist/app.js ./assets/js/app.js

✘ [ERROR] Could not resolve "flexsearch/dist/flexsearch.compact"

    assets/js/search.js:3:27:
      3 │ import { FlexSearch } from "flexsearch/dist/flexsearch.compact";
        ╵                            ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

  You can mark the path "flexsearch/dist/flexsearch.compact" as external to exclude it from the
  bundle, which will remove this error and leave the unresolved path in the bundle.

1 error
```

come back to fix this later.